app/views/holds/_form_inputs.html.erb

```diff
@@ -3,6 +3,6 @@
                                                     disabled: 'Not set', selected: 'Not set'),
                class: 'custom-select' %>
 <%= label_tag 'hold_expiration_date', 'Not needed after:' %>
-<%= date_field(:hold, :pickup_by_date) %>
+<%= date_field :hold, :pickup_by_date, value: default_pickup_by_date %>
```

Comment:
@cdmo Would you switch to `date_field_tag` instead of `date_field`?  The former was easier to use when creating holds. I have that changed in #119 PR but this is gonna merged before that one ;)

Reply:
So... I did change it at https://github.com/psu-libraries/myaccount/pull/128/files#diff-af88dbf8fe522d62f05ab051e404c44eR6

Sorry, maybe overlook for now though?

Reply:
ahhh sure 👍